docs/src/QuickStart/running-qat.md

```diff
@@ -0,0 +1,414 @@
+# Running QAT
+
+In the [next section](../UsingQAT/TargetingQIR.md) we will discuss the
```

Comment:
I'd prefer to call it "the section on targeting QIR" rather than "next section". Updating things like "next", "previous" etc. gets tedious and very easy to miss when we make changes to the docs structure.